helix-core/src/main/java/org/apache/helix/model/InstanceConfig.java

```diff
@@ -506,6 +508,16 @@ public void setMaxConcurrentTask(int maxConcurrentTask) {
     _record.setIntField(InstanceConfigProperty.MAX_CONCURRENT_TASK.name(), maxConcurrentTask);
   }
 
+  /**
+   * Get the target size of task thread pool
+   * @return the target size of task thread pool
+   */
+  public int getTargetTaskThreadPoolSize() {
```

Comment:
Could we add setters as well with some sort of validation logic?

Reply:
I agree.